okapi/services/caches/search/all.xml

```diff
@@ -169,6 +169,14 @@
         will set this parameter to <b>true</b>, then you will <b>always</b>
         receive an empty result.</p>
     </opt>
+    <opt name='powertrail_id'>
+        <p>BETA. Pipe-separated list of ints. If set, only caches which belong to 
+        <b>active</b> Power Trail which id is given will be included.</p>
```

Comment:
no właśnie - nie jestem pewien jak do tego podejść... 
z jednej strony fajnie by było pominąć odfiltrowywanie "aktywnych" ścieżek, ale wtedy ktoś sprytny mógłby podglądać ścieżkę przed publikacją - aby to obejść trzeba by rozbudować filtr tak by sprawdzać uprawnienia użytkownika. 
tak chyba będzie lepiej. 
nieopublikowanych keszy tez nie można chyba poprzez OKAPI pobrać.